packages/sdk/src/tests/storage.test.ts

```diff
@@ -104,18 +100,19 @@ describe('Storage tests', () => {
 
     expect(files.length).toBeGreaterThan(0);
     expect(files.every((f) => !!f.CID)).toBeTruthy();
+    cid = files[0].CID;
```

Comment:
If this test is skipped, the last test will fail, which is in a stand alone "describe" context. It would probably be better, to have a fixed cid in the `Generate IPFS link` test and not save this one.